docs/source/pages/operative_temperature_range.rst

```diff
@@ -0,0 +1,78 @@
+.. _operative_temperature_range:
+
+Operative temperature range
+===========================
+
+VPU temperature range
+*********************
+
+The visual processing unit - VPU (MyriadX) - is rated for industrial use and has operating 
+temperature -40C to 105C as seen `here <https://ark.intel.com/content/www/us/en/ark/products/125926/intel-movidius-myriad-x-vision-processing-unit-4gb.html#tab-blade-1-0-4>`__.
+Other components have higher max temperature, so 105C is the maximum working temperature for our devices.
+
+General use case
+****************
+
+In the general use case at a 25C ambient, the chip running at full power (i.e. worst-case heat generation) resides at around 70C maximum.
+This sets the thermal margin to a minimum of 35C, since the safe limit is 105C.
```

Comment:
add the calculation that this means that max (theoretical) ambient temp is 60 degrees C (maybe add the little degree simbol everywhere so it's clear it's degrees celsius)